components/View/FormattedDate/FormattedDate.js

```diff
@@ -10,7 +10,7 @@ const propTypes = {
     /**
      * Timestamp
      */
-    date: PropTypes.oneOfType([
+    value: PropTypes.oneOfType([
```

Comment:
Did you check the backward compatibility for this?
I created a HackedFormatted date so that it can accept both value/date.